### PR TITLE
[Virt] DPDK Checkup: Update image building scripts

### PR DESCRIPTION
**Pull request by RamLavi**
This PR is updating the scripts used to creation the rhel DPDK container disk image.

Version(s):
v4.15+

Issue:
https://issues.redhat.com/browse/CNV-35859

Link to docs preview:
https://70090--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-running-cluster-checkups#virt-building-vm-containerdisk-image_virt-running-cluster-checkups

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/virt-building-vm-containerdisk-image.adoc
Comment (orelmisan) [inline modules/virt-building-vm-containerdisk-image.adoc:132]: If `-firstboot=first-boot` is removed, then the section that creates the first boot script, should be removed as well.

Comment (RamLavi) [reply modules/virt-building-vm-containerdisk-image.adoc:132]: good catch! DONE

**Review by sjhala-ccs — COMMENTED** [review]
LGTM

**Comment by RamLavi** [comment]
> @RamLavi Are you waiting for QE review on this? Otherwise I can merge the PR.

Thanks for the review!
There are some technical issues we need to make sure before we merge this, moving to draft in the meanwhile

**Comment by RamLavi** [comment]
checked new image on the DPDK-checkup. works.
/hold cancel

**Review by orelmisan — APPROVED** [review]
Thank you

**Comment by RamLavi** [comment]
@sjhala-ccs please notice line 147 looks strange in the output. Any suggestions on how to fix it?

**Comment by bell-lev** [comment]
/lgtm
Tested.

**Comment by sjhala-ccs** [comment]
@RamLavi Is this ready to be merged?

**Comment by RamLavi** [comment]
> @RamLavi Is this ready to be merged?

yes please

**Pull request merged**
